src/RadioGroup.js

```diff
@@ -12,7 +12,7 @@ class RadioGroup extends React.Component {
         }),
         container: PropTypes.string,
         name: PropTypes.string.isRequired,
-        onChange: PropTypes.func.isRequired,
+        onChange: PropTypes.func,
```

Comment:
seriously?! Either you use react, or you don't...

I understand it's nice to be generic, but I think there's a limit to that lol